payjoin-cli/src/db/v2.rs

```diff
@@ -252,7 +281,10 @@ impl Database {
     pub(crate) fn get_inactive_recv_session_ids(&self) -> Result<Vec<(SessionId, u64)>> {
         let conn = self.get_connection()?;
         let mut stmt = conn.prepare(
-            "SELECT session_id, completed_at FROM receive_sessions WHERE completed_at IS NOT NULL",
+            "SELECT r.session_id, e.created_at 
+             FROM receive_sessions r 
+             JOIN receive_session_events e ON r.completed_event_id = e.id 
+             WHERE r.completed_event_id IS NOT NULL",
```

Comment:
I don't think this join is neccecary. A session is active until completed_event_id is not null.